lib/plugins/aws/invokeLocal/index2.test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
Tests should be placed in https://github.com/serverless/serverless/blob/master/test/unit/lib/plugins/aws/invokeLocal/index.test.js and replace old tests